### PR TITLE
Add react to user badge

### PR DESCRIPTION
**Pull request by eabousaif**
#### How to Ninja test this (screenshots are really helpful)

screen-shot-2020-02-26-at-9-55-56-am

### CODE REVIEW

File: app/pb_kits/playbook/pb_user_badge/badges/_badges.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+.veteran-st1{fill:#FFFFFF;}
+.veteran-st2{fill:#BB2432;}
+
+
+.st0{fill:url(#SVGID_1_);}
+.st1{fill:url(#SVGID_2_);}
+.st2{fill:url(#SVGID_3_);}
+.st3{fill:url(#SVGID_4_);}
+.st4{fill:url(#SVGID_5_);}
+.st5{fill:url(#SVGID_6_);}
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_user_badge/badges/_badges.scss]: Why is this required? Do we need to generate a new SVG?

Comment (eabousaif) [reply app/pb_kits/playbook/pb_user_badge/badges/_badges.scss]: Removed this file and added styles back to the svg files
